src/sentry/incidents/endpoints/organization_alert_rule_activations.py

```diff
@@ -0,0 +1,54 @@
+from django.db.models import Q
+from drf_spectacular.utils import extend_schema
+from rest_framework.request import Request
+from rest_framework.response import Response
+
+from sentry.api.api_owners import ApiOwner
+from sentry.api.api_publish_status import ApiPublishStatus
+from sentry.api.base import region_silo_endpoint
+from sentry.api.paginator import OffsetPaginator
+from sentry.api.serializers import serialize
+from sentry.api.serializers.models.alert_rule_activations import AlertRuleActivationsSerializer
+from sentry.apidocs.constants import RESPONSE_FORBIDDEN, RESPONSE_NOT_FOUND, RESPONSE_UNAUTHORIZED
+from sentry.apidocs.examples.metric_alert_examples import MetricAlertExamples
+from sentry.apidocs.parameters import GlobalParams, MetricAlertParams
+from sentry.apidocs.utils import inline_sentry_response_serializer
+from sentry.incidents.endpoints.bases import OrganizationAlertRuleEndpoint
+
+
+@extend_schema(tags=["Alerts"])
+@region_silo_endpoint
+class OrganizationAlertRuleActivationsEndpoint(OrganizationAlertRuleEndpoint):
+    owner = ApiOwner.ISSUES
```

Comment:
Are we not owning this?

Reply:
There's no owners for Alerts&Notifications? 😬 

I just copied what was in the existing alert_rule_index lol

Reply:
Please create a new value in the enum for ALERTS_AND_NOTIFICATIONS and assign it to your github team group name. @saponifi3d is working on fixing the ownerships